addons/mail/static/src/components/composer/composer.scss

```diff
@@ -225,46 +87,8 @@
 .o_Composer_textInput {
     appearance: none;
     outline: none;
-    background-color: $white;
```

Comment:
I removed the following css properties from this component and added the right classes to the `ComposerTextInput` component : 
https://github.com/odoo/odoo/pull/87438 (required)